src/olympia/amo/log.py

```diff
@@ -671,6 +671,21 @@ class REVIEWER_REPLY_VERSION(_LOG):
     review_queue = True
 
 
+class APPROVALNOTES_CHANGED(_LOG):
```

Comment:
Yeah I did not get what that list was about. So, should I add both `APPROVALNOTES_CHANGED` and `SOURCECODE_UPLOADED`?